src/integration/exchange/controllers/exchange.controller.ts

```diff
@@ -82,6 +84,8 @@ export class ExchangeController {
   @ApiExcludeEndpoint()
   @UseGuards(AuthGuard(), new RoleGuard(UserRole.ADMIN), UserActiveGuard)
   async trade(@Param('exchange') exchange: string, @Body() { from, to, amount }: TradeOrder): Promise<number> {
+    if (DisabledProcess(Process.EXCHANGE_TRADE)) throw new BadRequestException('Safety module deactivated');
+
```

Comment:
I would just write `throw new BadRequestException('Process disabled')` as we don't know if it really is the safety module